### PR TITLE
Ride Status Updates

### PR DESCRIPTION
**Pull request by hmesander**
____ Wrote Tests
_@hmesander_ Implemented
____ Reviewed
# Neccesary checkmarks:
- [] All Tests are Passing

- [x] The code will run locally

## Type of change
- [x] New feature
- [] Bug Fix

# Implements/Fixes:
* Adds Lyft API calls from frontend app to retrieve ride status updates on specified intervals and displays the status to the user.
* Adds RidePage component, which user is directed to once they confirm the cost estimate of the Lyft ride.
closes #

# Check the correct boxes

- [x] This broke nothing
- [] This broke some stuff
- [] This broke everything

# Testing Changes
- [x] No Tests have been changed
- [] Some Tests have been changed
- [] All of the Tests have been changed(Please describe what in the world happened)

# Checklist:

- [] My code has no unused/commented out code
- [x] I have reviewed my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [] I have fully tested my code

# Please Include a link to a gif of your feelings about this branch
Link:

### CODE REVIEW

File: src/components/RidePage.js
Comment (zdcthomas) [inline src/components/RidePage.js]: javascript object can use method notation for keys

**Review by zdcthomas — DISMISSED** [review]
it looks good, but i'm gonna wait until you seen the page to approve

**Review by jude-lawson — APPROVED** [review]

**Pull request merged**
